scripts/workspace-conversion/workspace-to-dataset.py

```diff
@@ -302,6 +302,7 @@ def generate_catalog_metadata(workspace):
         "dct:title": wsAttributes.pop("library:datasetName", None),
         "dct:version": wsAttributes.pop("library:datasetVersion", None),
         "dct:description": wsAttributes.pop("library:datasetDescription", None),
+        "dct:modified": datetime.now().strftime("%Y-%m-%dT%H:%M:%S"),
```

Comment:
Is there a reason to not use `workspace["workspace"]["lastModified"]` here? That seems like it contains the last modified date for the workspace.

Reply:
I thought this would be the time the catalog entry was last modified, not the time the workspace/underlying data was last modified.

Reply:
Talked offline, moved.